### DISCUSSION TITLE
Geomap supports Opentsdb?

### DISCUSSION THREAD

**Discussion by ylin30** [grafana/grafana#61000] [Dashboards, Panels & Visualizations]
Hi, does geomap supports Opentsdb time series query? I tried both latest grafana (v9.3.2) and v8.1.5 and can't figure it out. 

Please refer to the discussion in community [here]( https://community.grafana.com/t/how-to-use-geomap-with-opentsdb/78951) for more details.

thx

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). It looks like you got help in the community forum. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a complete solution and would like to request a feature to the geomap.
